users.go

```diff
@@ -582,3 +582,113 @@ func (s *UsersService) DeleteEmailForUser(uid int, eid int, options ...OptionFun
 
 	return s.client.Do(req, nil)
 }
+
+// ImpersonationToken represents an ImpersonationToken.
+//
+// GitLab API docs: https://docs.gitlab.com/ce/api/users.html#get-all-impersonation-tokens-of-a-user
+type ImpersonationToken struct {
+	ID        int        `json:"id"`
+	Active    *bool      `json:"active"`
+	Token     *string    `json:"token"`
+	Scopes    *[]string  `json:"scopes"`
+	Revoked   *bool      `json:"revoked"`
+	Name      *string    `json:"name"`
+	CreatedAt *time.Time `json:"created_at"`
+	ExpiresAt *time.Time `json:"expires_at"`
+}
+
+// GetAllImpersonationTokensOptions represents the available GetAllImpersonationTokens() options.
+//
+// GitLab API docs:
+// https://gitlab.com/gitlab-org/gitlab-ce/blob/9-0-stable/doc/api/users.md#get-all-impersonation-tokens-of-a-user
+type GetAllImpersonationTokensOptions struct {
+	State *string `url:"state,omitempty" json:"state,omitempty"`
+}
+
+// Get all impersonation tokens of a user.
+//
+// GitLab API docs:
+// https://gitlab.com/gitlab-org/gitlab-ce/blob/9-0-stable/doc/api/users.md#get-all-impersonation-tokens-of-a-user
+func (s *UsersService) GetAllImpersonationTokens(uid int, opt *GetAllImpersonationTokensOptions, options ...OptionFunc) ([]*ImpersonationToken, *Response, error) {
```

Comment:
I followed the pattern 'func (s *UsersService) DeleteEmailForUser(uid int,...', because 'uid' is a acronym of 'user_id'. Why you judge that 'user' has more consistency?